src/wwwroot/services/settings.js

```diff
@@ -231,5 +232,25 @@
         url: RELAY_CONFIG.cuitServiceBaseUrl + '/taxinfo/by-cuit/' + cuit
       });
     }
+
+    function resubscribeEmailAddress (subscriberEmail, reason, domainName, onExpectedError) {
```

Comment:
A little thing, space between function name and params.

Reply:
```suggestion
    function resubscribeEmailAddress(subscriberEmail, reason, domainName, onExpectedError) {
```

Good catch @ms-darianbenito! you can make it easy to understand using that "suggestion" GitHub's feature:

![image](https://user-images.githubusercontent.com/1157864/103897251-03adad00-50d2-11eb-81cc-ae5c320a7e4a.png)